app/src/main/java/com/moviefiend/torian/moviefiend/MovieListFragment.java

```diff
@@ -50,24 +50,24 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
     // <editor-fold desc="Loader">
     @Override
-    public Loader<ArrayList<MoviesResponse.MovieInfo>> onCreateLoader(int id, Bundle args) {
-        return new MovieInfoLoader(getActivity(), UrlHelper.getNowPlayingUrl(BuildConfig.TMDB_API_KEY));
+    public Loader<ArrayList<MovieInfo>> onCreateLoader(int id, Bundle args) {
```

Comment:
@ifeins Hmmm. I usually (my own style) add an empty line before and always after a chain method call (builders, streams and such) to make it more readable.
wdyt?